docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+services:
+  db:
+    image: mysql:5.6
+    environment:
+      MYSQL_ROOT_PASSWORD: swordfish
```

Comment:
You may want to make use of environment variables or secrets so you don't push sensitive data to Github. Not required for this task though. Check out [env variables](https://docs.docker.com/compose/environment-variables/) and [secrets](https://docs.docker.com/engine/swarm/secrets/#defining-and-using-secrets-in-compose-files)